### ISSUE TITLE
[needs verification] Building hours update for RNS

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#3097]
From an email; we should verify this. 

## Before:

```yaml
name: RNS
subtitle: Regents Hall of Natural Sciences
category: Academia
image: regents-hall

schedule:
  - title: Hours
    hours:
      - {days: [Mo, Tu, We, Th, Fr, Sa, Su], from: '7:00am', to: '10:00pm'}

breakSchedule:
  fall: []
  thanksgiving: []
  winter: []
  interim: []
  spring: []
  easter: []
  summer: []
```

## After:

```yaml
name: RNS
subtitle: Regents Hall of Natural Sciences
category: Academia
image: regents-hall

schedule:
  - title: Hours
    hours:
      - {days: [Mo, Tu, We, Th, Fr, Sa, Su], from: '7:00am', to: '12:00am'}

breakSchedule:
  fall: []
  thanksgiving: []
  winter: []
  interim: []
  spring: []
  easter: []
  summer: []
```

**Comment by drewvolz** [comment]
I would think we’d have different weekday vs weekend hours

**Comment by hawkrives** [comment]
According to my copy of Tim's Building Door Lock Schedule™, last updated 2017/09/15, RNS is indeed open from 8am-12am every day of the week, including weekends.

**Comment by drewvolz** [comment]
Then the first thing to pop up when I google “what are regents hall hours” is incorrect. Thanks for checking Tim’s schedule. ![GitHawk Upload by drewvolz](https://i.imgur.com/aim3k5G.jpg)

**Comment by hawkrives** [comment]
… actually, Tim's schedule doesn't make sense then – how are you supposed to get to your 8am if the doors are locked before 8?

**Comment by hawkrives** [comment]
I'ma go ask.

**Comment by hawkrives** [comment]
Okay, hang on. I found more tabs in the spreadsheet.

- <kbd>Acad. Yr. Posted Bldg Hrs</kbd> says 8am-12am.
- <kbd>Acad. Yr. Published Bldg Hrs</kbd> says 7am-12am.
- <kbd>Acad. Yr. Elec. Bldg Hrs</kbd> says 7am-11:59pm.

I'm going to go inquire about the difference between "Posted" and "Published".

---

I think the correct time is 7am-12pm, 7 days a week, as that's what's listed on my Published tab.

**Comment by hawkrives** [comment]
Okay. I've talked with Tim.

My "Posted Hours" tab should have been hidden; I guess Gmail ignored that. It hasn't been updated since 2017 September.

The "Published Hours" tab has been updated since the last hours email was sent out; it is now 8am-12am.

It's set to 8am so that "if something goes funky [with the auto-unlock at 7am], we have an hour to fix it before people start pounding on the doors".

> the doors _unlock_ at 7am, but the posted hours are 8am

---

I vote that we just follow the published hours and put 8am-12am in the app.
